src/NeoSharp.Core/Persistence/ByteArrayExtensionMethods.cs

```diff
@@ -7,7 +8,14 @@ public static class ByteArrayExtensionMethods
         public static byte[] BuildKey(this byte[] hash, DataEntryPrefix dataEntryPrefix)
```

Comment:
`public static byte[] BuildKey(this DataEntryPrefix dataEntryPrefix, params byte[] hash )`
?

Reply:
that could work and be a good idea.... going to try